### ISSUE TITLE
10.8.2 Alpha Enhancement for Genre presentation:

### ISSUE THREAD

**Issue by MRDJellyfin** [jellyfin/jellyfin#6916]
### Please describe your bug

I don't know if the same thing happens in other Languages ​​but in Windows 10 Pro in Spanish when the option to select the GENRE is displayed in movies, there are some that are duplicated (ALMOST ALL), since they are both in Spanish and in English. There is no translation for Biography in Spanish "BIOGRAFIA", the translation of Suspense is wrong or does not have to be "SUSPENSO".

![image](https://user-images.githubusercontent.com/82923908/143724993-375d4a75-41df-47d2-96ac-5a08f0c44901.png)

### Jellyfin Version

Other

### Environment

```markdown
- OS:
- Virtualization:
- Clients:
- Browser:
- FFmpeg Version:
- Playback Method:
- Hardware Acceleration:
- Plugins:
- Reverse Proxy:
- Base URL:
- Networking:
- Storage:
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by crobibero** [comment]
I don’t think this is a bug.. the genre list is built using your media and the selected metadata providers

**Comment by MRDJellyfin** [comment]
I understand that and sorry for my ignorance, but it would not be then that a conversion table existed in the program to show the genre according to the language selected by one. Which, to my poor understanding, would be more or less similar to the one that must exist in the program due to the audio, since the conversion of the metadata contained in the movie file is done and points out in the description of the movie, as audio the complete word of the language and in Spanish (in my case Spanish, I don't know if it does the same for other languages), in almost all languages, since in the following that I have found so far there is no conversion: Chi (Chinese=CHINO), Fre (French=FRANCES), Ger (German=ALEMAN), Geo (Georgian=GEORGIANO), Ice (Icelandic=ISLANDES), May (Malay=MALAYO) and Tgl (Tagalog=TAGALO). For which if I have found the correct presentation in Spanish are: "DANES" (Danish), "ESPAÑOL" (Spanish), "FILIPINO" (Filipino), "ENGLISH", "ITALIANO" (Italian), "RUSO" (Russian ), "INDONESIO" (Indonesian), "COREANO" (KOREAN), "JAPONES" (Japanese), "PORTUGUES" (Portuguese), "SUECO" (Swedish), and "TAILANDES" (Thai). And once again, sorry if I've written something silly, the only thing I'm looking for is for the program to get better and better
